toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -30,7 +30,13 @@ const BindMountPointFlags = unix.MS_BIND | unix.MS_MGC_VAL
 
 // FileToCopy represents a file to copy into a chroot using AddFiles. Dest is relative to the chroot directory.
 type FileToCopy struct {
-	Src         string
+	// The source file path.
```

Comment:
So, we have the old and new config structs  (json vs. yaml). We tend to use the json versions of those structs in the imager code a bunch. I'm not sure what feels least odd here, why re-implement the same data structure twice? But... I believe we also generally don't directly import the image customizer types outside the customizer tool...

(No real opinion here, just feels a bit odd somehow. Probably just a consequence of old design choices we live with now)